src/Controls/src/Core/Registrar.cs

```diff
@@ -384,12 +384,14 @@ public static void RegisterEffect(string resolutionName, string id, Type effectT
 		}
 
 		/// <include file="../../docs/Microsoft.Maui.Controls.Internals/Registrar.xml" path="//Member[@MemberName='RegisterAll']/Docs" />
+		[Obsolete]
 		public static void RegisterAll(Type[] attrTypes, IFontRegistrar fontRegistrar = null)
```

Comment:
Made the RegisterAll methods obsolete as these are probably not the droids users are looking for.